code/modules/mob/living/carbon/human/species/necromorph/exploder.dm

```diff
@@ -12,6 +12,7 @@
 	total_health = 85	//It has high health for the sake of making it a bit harder to destroy without targeting the pustule. Exploding the pustule is always an instakill
 	biomass = 70
 	mass = 50
+	species_flags = SPECIES_FLAG_NO_PAIN | SPECIES_FLAG_NO_MINOR_CUT | SPECIES_FLAG_NO_POISON  | SPECIES_FLAG_NO_BLOCK | SPECIES_FLAG_NO_SLIP
```

Comment:
This is not a fix, it was working as intended. Revert these changes to species flags.

I pay careful attention to the flags on every new necromorph, their existing state was designed, not an accident